### PR TITLE
adding edrixs recipe

### PR DESCRIPTION
**Pull request by kbeeperez**
implements: 
https://github.com/nsls-ii-forge/arpack-mpi-feedstock
https://github.com/nsls-ii-forge/edrixs-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/arpack-mpi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  url: https://github.com/opencollab/arpack-ng/archive/{{ version }}.tar.gz
+  sha256: ada5aeb3878874383307239c9235b716a8a170c6d096a6625bfd529844df003d
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - cmake
+    - openmpi
```
Comment (kbeeperez) [inline recipes/arpack-mpi/meta.yaml:17]:
```suggestion
    - openmpi
    - lapack
```

**Comment by kbeeperez** [comment]
@conda-forge/staged-recipes Hello, I am receiving this error https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=356884&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1745

I am I missing something in the dependencies for `edrixs` recipe?

**Comment by mrakitin** [comment]
Is https://github.com/conda-forge/arpack-feedstock something that we can use as a dependency instead of `arpack-mpi`? Could you try adding it as a dependency of `edrixs`, and remove the `arpack-mpi` recipe?

**Comment by mrakitin** [comment]
> Is https://github.com/conda-forge/arpack-feedstock something that we can use as a dependency instead of `arpack-mpi`? Could you try adding it as a dependency of `edrixs`, and remove the `arpack-mpi` recipe?

Looks like it does not have the mpi support. Maybe we can use https://github.com/conda-forge/arpack-feedstock/blob/4339d6fbb352c85afc06ea1274cc1768874f6e22/recipe/meta.yaml#L18-L30 as an example of how to add the required libraries, and eventually make it a part of the `arpack` feedstock...

**Comment by kbeeperez** [comment]
Attn @mrakitin

**Comment by kbeeperez** [comment]
@conda-forge/staged-recipes I have tried adding `openblas` and `libopenblas` to different parts of `edrixs` meta.yaml  but it is not working?

**Comment by isuruf** [comment]
Please read our docs on how to use BLAS libraries. Also, please don't add `arpack-mpi` when there's `arpack-feedstock`. If you need MPI support for arpack please add it there and not as a separate recipe.

**Pull request closed without merging**